sphinx/source/users/issue_commands.rst

```diff
@@ -75,3 +75,66 @@ To guarantee that their request is successfully committed to the ledger, a user
     }
 
 In this example, the ``result`` field indicates that the request was executed at ``30`` (``commit``) was in term ``2``, the same term that the ``LOG_record``. Moreover, the ``global_commit`` (``31``) is now greater than the ``commit`` version. The ``LOG_record`` request issued earlier was successfully committed to the ledger.
+
+Transaction receipts
+--------------------
+
+Once a transaction has been committed, it is possible to get a receipt for it. That receipt can later be checked against either a CCF service, or offline against the later, to prove that the transaction did happen at a particular version.
```

Comment:
"offline against the later" -> "offline against the ledger"?

Reply:
Yes, sorry, fail...